README.md

```diff
@@ -157,7 +179,7 @@ deploy your dApp.
 You can use one of the deployed test accounts and associated private key with
 MetaMask. If you use the same MetaMask accounts on fresh local networks such as
 Hardhat Node, Foundry Anvil or sapphire-dev docker image, don't forget to
-*clear your account's activity* each time or manually specify the correct
+_clear your account's activity_ each time or manually specify the correct
```

Comment:
Stick to either asterisks or underlines. Since the parameters above use `**` I'd stick to aseterisks. This is probably bold not italic right?

```suggestion
**clear your account's activity** each time or manually specify the correct
```